EventView-1/pom.xml

```diff
@@ -23,19 +23,24 @@
     <dependencies>
         <dependency>
             <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-data-jpa</artifactId>
+            <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-web</artifactId>
+            <artifactId>spring-boot-starter-security</artifactId>
+            <version>2.2.7.RELEASE</version>
         </dependency>
```

Comment:
inconsistent versions across pom for boot dependency , correct it or remove <version> entirely to use spring boot parent version from top.

Reply:
okay will fix it